### PR TITLE
Fixed version compatibility error when importing speedster

### PR DESCRIPTION
**Pull request by arianGh1**
when I wanted to run the cell containing `from speedster import optimize_model` on Google Colab this was the error that I got:  

`ContextualVersionConflict: (protobuf 3.19.6 (/usr/local/lib/python3.8/dist-packages), Requirement.parse('protobuf<4,>=3.20.2'), {'onnx'})`  

so I added this cell to the notebook and the problem was fixed:
```
!pip install virtualenv
!virtualenv -p python3 newenv   
!pip install protobuf==3.12.2 speedster onnx==1.8.0
```

### CODE REVIEW

**Comment by diegofiori** [comment]
Hello @arianGh1! Thanks for your contribution! Could you just clear the notebook output before we merge this PR?

**Comment by arianGh1** [comment]
Sure

**Comment by arianGh1** [comment]
@diegofiori  Did it

File: notebooks/speedster/tensorflow/Accelerate_Tensorflow_ResNet50_with_Speedster.ipynb [hunk 1]
```diff
@@ -115,7 +127,7 @@
       "source": [
         "In the following example we will try to optimize a standard resnet50 loaded directly from keras.\n",
         "\n",
-        "Speedster can accelerate neural networks without loss of a user-defined precision metric, e.g. accuracy, or can achieve faster acceleration by applying more aggressive optimization techniques, such as pruning and quantization, that may have a negative impact on the selectic metric. The maximum threshold value for accuracy loss is determined by the metric_drop_ths parameter. Read more in the [docs](https://docs.nebuly.com/modules/speedster/getting-started).\n",
+        "Speedster can accelerate neural networks without loss of a user-defined precision metric, e.g. accuracy, or can achieve faster acceleration by applying more aggressive optimization techniques, such as pruning and quantization, that may have a negative impact on the selectic metric. The maximum threshold value for accuracy loss is determined by the metric_drop_ths parameter. Read more in the [docs](https://nebuly.gitbook.io/nebuly/nebullvm/get-started).\n",
```
Comment (diegofiori) [inline notebooks/speedster/tensorflow/Accelerate_Tensorflow_ResNet50_with_Speedster.ipynb:130]: One more thing. Could you keep here the updated link to the docs? I mean the https://docs.nebuly.com/modules/speedster/getting-started one

**Comment by arianGh1** [comment]
@diegofiori Updated the link

**Pull request merged**